src/main/java/lamball/TaskList.java

```diff
@@ -0,0 +1,207 @@
+package lamball;
+
+
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+import lamball.task.Deadline;
+import lamball.task.Event;
+import lamball.task.Task;
+import lamball.task.ToDo;
+
+
+
+/**
+ * A class that handles the to-do list operations.
+ *
+ * @author ongzhili
+ */
+public class TaskList {
+    protected ArrayList<Task> tasks;
+
+    private String lastDoneTask;
+
+
+    /**
+     * Constructor for TaskList
+     *
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+        this.lastDoneTask = "";
+    }
+
+    /**
+     * Returns last done task.
+     *
+     * @return last done task in String format.
+     */
+    public String getLastDoneTask() {
+        return this.lastDoneTask;
+    }
+
+    private void printList(ArrayList<Task> lst) {
```

Comment:
Perhaps we can spell out the argument name fully, lst can look like 1st to some. :)